da/index.html

```diff
@@ -34,7 +34,8 @@ <h2 class="welcome-subtitle">Parseur de pseudo-code</h2>
     <header class="topbar">
         <h1 class="topbar-title">Editeur de diagrammes d'actions (pseudo-code)</h1>
         <nav class="topbar-menu">
-            <a href="#" class="menu-item" id="buttonNew">Nouveau</a>
+			<a href="#" class="menu-item" id="buttonNew">Nouveau</a>
+			<a href="#" class="menu-item" id="buttonSaveInCache">Sauvegarder</a>
```

Comment:
Pay attention to the indentation please :)

Reply:
![image](https://user-images.githubusercontent.com/16562647/35779211-980d5c78-09c9-11e8-9acf-9c3ca8399a32.png)
This is embarrassing. On VisualCode, there are no intentation problems.